tests/testthat/test-describe_distribution.R

```diff
@@ -351,3 +351,34 @@ test_that("errors on invalid column names (reserved word)", {
     regex = "Following variable names are reserved"
   )
 })
+
+# multiple centralities
+test_that("multiple centralities work", {
```

Comment:
Maybe you were already planning to add them, but can we also test with `ci = 0.95` for instance to check that the output with multiple CI columns is fine?